Libs/DICOM/Core/Testing/Cpp/ctkDICOMSchedulerTest1.cpp

```diff
@@ -151,7 +151,6 @@ int ctkDICOMSchedulerTest1(int argc, char* argv[])
       sopIstanceUID, QThread::LowPriority, QStringList("Test"));
   }
 
```

Comment:
```suggestion
CHECK_INT(scheduler.numberOfRunningJobs(), numberOfImages);
```

this should be safe and the check return true. See https://github.com/commontk/CTK/pull/1255#issuecomment-3151989787 for more info